source/developers-guide/plugin-configuration/index.md

```diff
@@ -20,25 +20,36 @@ the configuration options for plugins. Shopware delivers a big amount of helper
 configuration fields in backend.
 
 ## Creating forms
-To create a simple plugin configuration we can use the function `Form()`
+Backend plugin configuration can be extended by Resources/config.xml file.
```

Comment:
A plugin configuration can be created with a Resources/config.xml file.

Reply:
d